### DISCUSSION TITLE
[BUG] Unable to delete discussion category

### DISCUSSION THREAD

**Discussion by vivi90** [community/community#3390] [Discussions]
I want to delete a freshly created empty discussion category but i get an error page, what says:
```
This page is taking too long to load.

Sorry about that. Please try refreshing and contact us if the problem persists.
```

The discussion board: https://github.com/HTWK-18INB-1/harae/discussions
The url of the error page: https://github.com/HTWK-18INB-1/harae/discussions/categories/32593092
The category, i wanted to delete: `test`

All other things seems to work properly.

**Comment by PranamBhat** [comment]
Hello @vivi90 

Thank you for bringing this out.

Seems like your session has been expired. Try refresh (**Ctrl + F5**) and deleting the same.

**Reply by gilgongo** [reply]
See also https://github.com/github/feedback/discussions/3352
